Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocMultilineTextValue.css

```diff
@@ -7,36 +7,31 @@ span.bocMultilineTextValue
   position: relative;
   display: inline-block;
   vertical-align: middle;
-  overflow: hidden;
 }
 
 .bocMultilineTextValue.readOnly
 {
   min-height: 0;
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
 }
 
 .bocMultilineTextValue.disabled
 {
 }
 
+span.bocMultilineTextValue .content,
+span.bocMultilineTextValue textarea
+{
+  box-sizing: border-box;
+}
+
 span.bocMultilineTextValue .content
 {
   position: absolute;
   display: block;
-  left: 0;
-  right: 6px;
-  width: auto;
-  top: 0;
-  bottom: 0.5em;
+  width: inherit;
```

Comment:
width:inherit is not necessary if replaced with width:100% (is this better?). height:auto was now replaced by height:100% in order to fill the space of the parent so that the textarea is neither smaller nor larger than the parent since without it, it wouldn't fit exactly (larger in Firefox, smaller in Chrome/Edge), display: block alone is not enough.

Reply:
sounds good.